components/avatar/avatar.css

```diff
@@ -0,0 +1,81 @@
+/* @import url('../../components/root.css'); */
+
+@import url('../../components/framer.css');
+
+
+.avatar-space-default{
+    padding: 4.8px;
+ }
+ 
+ .avatar-space-group{
+     padding: 0px;
+     margin-left: -2rem;
+  }
+ 
+ 
+ .avatar-lg{
+  width: 10rem;   
+ }
+ .avatar-md{
+     width: 7rem;   
+ }
+ .avatar-sm{
+     width: 5rem;   
+ }
+ .avatar-ex-sm{
+     width: 3rem;
+ }
```

Comment:
code looks ready to merge, just a suggestion naming can be shorten.

Reply:
@Sahil25061999 I appreciate your suggestion can you elaborate more I mean what naming part can be reduced, here I am already using shorthand naming for extra-small.